libkbfs/crypto_client.go

```diff
@@ -10,110 +10,54 @@ import (
 	"github.com/keybase/client/go/libkb"
 	"github.com/keybase/client/go/logger"
 	keybase1 "github.com/keybase/client/go/protocol"
-	rpc "github.com/keybase/go-framed-msgpack-rpc"
 	"golang.org/x/net/context"
 )
 
-// CryptoClient implements the Crypto interface by sending RPCs to the
-// keybase daemon to perform signatures using the device's current
-// signing key.
+// CryptoClientProvider should match keybase1.CryptoClient for signing
+// and unboxing.
+type CryptoClientProvider interface {
```

Comment:
Wait, this is literally identical to `keybase1.CryptoClient`.  Why are we redefining it?  Are you trying to avoid importing the protcol files somewhere?